include/securityhub_integration

```diff
@@ -42,24 +42,43 @@ checkSecurityHubCompatibility(){
 
 resolveSecurityHubPreviousFails(){
   # Move previous check findings RecordState to ARCHIVED (as prowler didn't re-detect them)
+    SH_TEMP_FOLDER="$PROWLER_DIR/SH-$ACCOUNT_NUM"
+    if [[ ! -d $SH_TEMP_FOLDER ]]; then
+      # this folder is deleted once the security hub update is completed
```

Comment:
where is the folder deleted? I don't see a `rm` here.

Reply:
That was my mistake, I forgot to add that line since I was doing everything in a container, there was nothing left to remind me. I've added an additional commit that adds `rm -rf "$SH_TEMP_FOLDER"` to the end of the `resolveSecurityHubPreviousFails(){` function.